docs/resources/aws_config_delivery_channel.md

```diff
@@ -26,52 +29,51 @@ An `aws_config_delivery_channel` resource block declares the tests for a single
 
 The following examples show how to use this InSpec audit resource.
 
-### Test if the recorder is active and recording.
+### Test how frequent the channel writes configuration changes to the s3 bucket.
 
-    describe aws_config_delivery_channel(recorder_name: 'my-recorder') do
-      it { should be_recording }
+    describe aws_config_delivery_channel(channel_name: 'my-recorder') do
+      its(delivery_frequency) { should be > 3 }
```

Comment:
If the property has been changed to be `delivery_frequency_in_hours`, need to update it here too.